spec/sidekiq-instrument/server_middleware_spec.rb

```diff
@@ -32,16 +32,20 @@
         Sidekiq[:max_retries] = 0
       end
 
-      it 'increments StatsD dequeue counter' do
+      it 'increments StatsD dequeue and success counters' do
         expect do
           MyWorker.perform_async
-        end.to trigger_statsd_increment('shared.sidekiq.default.MyWorker.dequeue')
+        end.to trigger_statsd_increment('shared.sidekiq.default.MyWorker.dequeue') &&
+               trigger_statsd_increment('shared.sidekiq.default.MyWorker.success')
       end
 
-      it 'increments DogStatsD dequeue counter' do
+      it 'increments DogStatsD dequeue and success counters' do
         expect(
```

Comment:
nit: can lines 43 to 48 be shortened to:
```
  expect(Sidekiq::Instrument::Statter.dogstatsd).to receive(:increment)
    .with('sidekiq.dequeue', expected_dog_options).once
    .and receive(:increment)
    .with('sidekiq.success', expected_dog_options).once
```

Reply:
I think this will break the pattern.

Reply:
ok, nvm.

Reply:
This code isn't quite working and I couldn't get it to work with a few attempted changes, so would prefer to leave it